Project.toml

```diff
@@ -1,22 +1,23 @@
+authors = ["Samuel Omlin", "Ludovic Räss", "Ivan Utkin"]
 name = "ImplicitGlobalGrid"
 uuid = "4d7a3746-15be-11ea-1130-334b0c4f5fa0"
-authors = ["Samuel Omlin", "Ludovic Räss"]
-version = "0.11.1"
+version = "0.13.0"
+
+[compat]
+AMDGPU = "0.3.7"
+CUDA = "1, ~3.1, ~3.2, ~3.3, ~3.7.1, ~3.8, ~3.9, ~3.10, ~3.11, ~3.12"
+LoopVectorization = "0.12"
+MPI = "0.20"
```

Comment:
```suggestion
MPI = "0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.18, 0.19"
```

Reply:
I set MPI.jl it back to what is was until v0.20 is released, as else, we cannot run any tests anymore.